BAZProject/Helpers/Constans.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Constans.swift
+//  BAZProject
+//
+//  Created by victor manzanero on 07/11/22.
+//
+
+import Foundation
+
+class Constans {
```

Comment:
Tienes un typo aquí:
```suggestion
class Constants {
```